common/plane.h

```diff
@@ -0,0 +1,26 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include "float3.h"
+
+namespace rs2 {
+
```

Comment:
I assume you mean that the plane's equation is: ax+by+cz+d = 0?
Please add this as a comment here.